### ISSUE TITLE
Need new flag: "--discard_R2" (should work with and without -SE)

### ISSUE THREAD

**Issue by GabeAl** [knights-lab/shi7#44]
This would be useful with things like '-SE', which by default still keeps all the R2 reads (it merely disables *checking* whether reads are paired, but doesn't do anything about it if they are and the user just wants R1). This may be desirable sometimes, other times not. 

Similarly, if the user has paired end reads that don't stitch and uses --flash false, the "--discard_R2" flag would simply chuck the R2's after using them for adapter removal.

**Comment by RRShieldsCutler** [comment]
I've added this in my fork. The argument is `--drop_r2`. It works with `--flash False` or `-SE`, but does require the fastq filename to either 1) end with R1/R2.fastq, or 2) concurrent use of the `--strip_underscore True` argument.

It also removes the "R1" label, which makes matching to samples easier downstream.

I've tested it on a set of ~180 shallow shotgun fastq. It works, and was almost 30% faster than writing both R1 and R2. Note: using `--flash False` retains more total reads than `-SE`.

https://github.com/RRShieldsCutler/shi7/blob/master/bin/
